contributor-guides/for-staff.md

```diff
@@ -14,20 +14,23 @@ Here are some important things to figure out in the first couple of days:
 * Team Public Code chat - say hello, we're a welcoming bunch
 * Calendar - add your holidays to the team calendar
 * Manual of me - fill out your own and read the existing ones
-* Github - push a branch, make a pull request (maybe to improve this page!)
+* Oodo - We use ODOO for our internal chat tool as well as project management and customer relations management
+* Github - push a branch, make a pull request (maybe to improve this page!). You can request an invite to have our Github account added to the organizational account.
 * Splitwise - if you want to join the shared lunch groceries
 * Attend morning standup ritual that is in the team calendar
 * Attend sprint retrospective & planning - review the kanbans beforehand
 * Tentoo NMBRS ESS (employee self service) mobile app - to [request leave](../activities/staff-management/leave.md), [file expenses](../activities/staff-management/expense.md) and see your payslips
 
-Any of your new coworkers will be glad to help! <3
+Any of your new co-workers will be glad to help! <3
```

Comment:
What do you think @ElenaFdR?

Reply:
This is a distinction of American vs British English. We use American English at the Foundation for Public Code.